pattern-library/sass/patterns/_icons.scss

```diff
@@ -17,117 +17,114 @@
 @import 'icons-afontgarde';
 
 $edx-icons:
-linkedin-square
+angle-down
```

Comment:
Reviewers: I made this list alphabetical to make it easier to find if an icon exists.